charts/seed-bootstrap/dashboards/seed-deployments-replicas.json

```diff
@@ -0,0 +1,432 @@
+      "fieldConfig": {
+        "defaults": {},
+        "overrides": []
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 10,
+        "w": 24,
+        "x": 0,
+        "y": 4
+      },
+      "hiddenSeries": false,
+      "id": 10,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
+      "percentage": false,
+      "pluginVersion": "7.5.17",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "max(kube_deployment_status_replicas{deployment=\"$deployment_name\"}) without (instance, pod)",
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "{{namespace}}: current replicas",
+          "refId": "A"
+        },
+        {
+          "exemplar": true,
+          "expr": "min(kube_deployment_status_replicas_available{deployment=\"$deployment_name\"}) without (instance, pod)",
+          "hide": false,
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "{{namespace}}: available",
+          "refId": "B"
+        },
+        {
+          "exemplar": true,
+          "expr": "max(kube_deployment_status_replicas_unavailable{deployment=\"$deployment_name\"}) without (instance, pod)",
+          "hide": false,
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "{{namespace}}: unavailable",
+          "refId": "C"
+        },
+        {
+          "exemplar": true,
+          "expr": "min(kube_deployment_status_replicas_updated{deployment=\"$deployment_name\"}) without (instance, pod)",
+          "hide": false,
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "{{namespace}}: updated",
+          "refId": "D"
+        },
+        {
+          "exemplar": true,
+          "expr": "max(kube_deployment_spec_replicas{deployment=\"$deployment_name\"}) without (instance, pod)",
+          "hide": false,
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "{{namespace}}: desired",
+          "refId": "E"
+        }
+      ],
+      "thresholds": [],
+      "timeRegions": [],
+      "title": "Replicas",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets": null,
+        "mode": "time",
+        "name": null,
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "format": "short",
+          "label": null,
```

Comment:
Do you want to maybe limit the decimals on this graph, that way the Y-axis looks a little cleaner, seeing how the number of replicas is always going to be an integer.

```suggestion
          "decimals": 0,
          "format": "short",
          "label": null,
```
![Screenshot 2023-05-17 at 17 25 43](https://github.com/gardener/gardener/assets/9338170/37a5f261-ee3a-44a9-8afd-a16090534ff1)

Note that even though I added this suggestion in the JSON here, you probably want to do this in Plutono/Grafana and export the JSON again.